Tools/CISupport/ews-build/steps.py

```diff
@@ -1680,39 +1680,6 @@ def start(self):
         return None
 
 
-class ValidateChangeLogAndReviewer(shell.ShellCommand):
-    name = 'validate-changelog-and-reviewer'
-    descriptionDone = ['Validated ChangeLog and Reviewer']
-    command = ['python3', 'Tools/Scripts/webkit-patch', 'validate-changelog', '--check-oops', '--non-interactive']
```

Comment:
without this how is Merge-Queue validating reviewer for a PR which contains "Reviewed by XYZ" line in commit message and doesn't have explicit approval in GitHub UI?

Reply:
[class ValidateCommitMessage(steps.ShellSequence, ShellMixin):](https://github.com/WebKit/WebKit/blob/622f5bf389dec8cff2e4bf744b1e17a15c751edd/Tools/CISupport/ews-build/steps.py#L5004), although it looks like we don't check that the listed reviewer is actually a reviewer, so we need a step that does that.

Reply:
Oh yikes, I totally got this one wrong! I don't know why I thought `ValidateCommitterAndReviewer` used changelogs, but it doesn't We definitely need that step! I just need to have `ValidateCommitMessage` populate the `reviewer` field.

Reply:
To @aj062's point, https://bugs.webkit.org/show_bug.cgi?id=240718 will cover adding a check for reviewer status in the `ValidateCommitMessage` step.